lib/remote/consolehandler.cpp

```diff
@@ -88,6 +88,13 @@ bool ConsoleHandler::HandleRequest(
 
 	bool sandboxed = HttpUtility::GetLastParameter(params, "sandboxed");
 
+	ConfigObjectsSharedLock lock (std::try_to_lock);
```

Comment:
The instantiated class' header is missing: https://github.com/Icinga/icinga2/actions/runs/5750509316/job/15587428056?pr=9841#step:6:1625